internal/records/record_builder.go

```diff
@@ -739,11 +738,21 @@ func newUUIDSliceField(mem memory.Allocator, name string) (f *fieldBuilderFunc)
 	}
 	bd := b.(*array.BinaryDictionaryBuilder)
 	f.buildFunc = func(v reflect.Value) error {
-		return bd.Append(samples.ExtractLocationIDs(v.Interface().([]uuid.UUID)))
+		return bd.Append(ExtractLocationIDs(v.Interface().([]uuid.UUID)))
 	}
 	return
 }
 
+func ExtractLocationIDs(locs []uuid.UUID) []byte {
```

Comment:
This was copied to avoid importing `sample` package. Eventually `[]uuid.UUID` handling will go away together with this. Let's leave it here for now.